docs/installation-and-setup.md

```diff
@@ -133,6 +146,18 @@ return [
          * The directory where the temporary files will be stored.
          */
         'temporary_directory' => storage_path('app/backup-temp'),
+
+        /*
+         * The password to be used for archive encryption.
+         * Set to `null` to disable encryption.
+         */
+        'password' => env('BACKUP_ARCHIVE_PASSWORD'),
+
+        /*
+         * The encryption algorithm to be used for archive encryption.
+         * You can set it to `null` or `false` to disable encryption.
+         */
+        'encryption' => \ZipArchive::EM_AES_256,
```

Comment:
All changes except of this are only the copied real config as this document was out of sync.